src/olympia/editors/helpers.py

```diff
@@ -578,7 +579,9 @@ def get_actions(self, request, addon):
                                      'label': label}
             # An unlisted sideload add-on, which requests a full review, cannot
             # be granted a preliminary review.
-            if addon.is_listed or self.review_type == 'preliminary':
+            prelim_allowed = not waffle.flag_is_active(
+                request, 'no-prelim-review') and addon.is_listed
+            if prelim_allowed or self.review_type == 'preliminary':
```

Comment:
So if `review_type` is set to `'preliminary'` the option will still be present, even if the `no-prelim-review` flag is active ? That seems weird. Do we still need that action in some way?